prioritization_fees/src/account_priofees.rs

```diff
@@ -178,24 +190,41 @@ impl AccountPrioStore {
         let min_slot_to_retain = produced_block
             .slot
             .saturating_sub(self.number_of_slots_to_save as u64);
-        let cleanup_functor = |mut iter: RefMutMulti<'_, Pubkey, AccountPrio>| {
+        let cleanup_functor = |iter: &mut AccountPrio| {
             while let Some((k, _)) = iter.stats_by_slot.first_key_value() {
```

Comment:
you can use the BtreeMap.iter() which gives the entries sorted by key

Reply:
Yes that is why i get the first element compare and pop it out. In C++ i know how to iterate and remove multiple first elements. But in rust i cannot find the way.